facturacion_mexico/cfdi_recibidos/parsers/cfdi_recibido_parser.py

```
@@ -0,0 +1,252 @@
+class CFDIRecibidoParser:
+	"""
+	Parsea un XML CFDI recibido y extrae todos los datos fiscales.
+
+	Uso:
+	    parser = CFDIRecibidoParser(xml_bytes)
+	    data = parser.parse()
+
+	Lanza frappe.ValidationError si la versión no es 4.0 o el XML es inválido.
+	"""
+
+	def __init__(self, xml_content: bytes | str):
```

Ruff (0.15.13)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)